vowpalwabbit/core/src/reductions/generate_interactions.cc

```diff
@@ -206,6 +206,8 @@ std::shared_ptr<VW::LEARNER::learner> VW::reductions::generate_interactions_setu
     }
   }
 
+  if (options.was_supplied("automl")) { interactions_spec_contains_wildcards = true; }
```

Comment:
this in theory should be per ft_offset - not all of them have wildcards

thoughts @bassmang

Reply:
with the added challenge that the configs change ever so often